ci-scripts/request-translation.sh

```diff
@@ -2,7 +2,7 @@
 
 cd ..
 
-export SLACK_CHANNEL='#doctools-publish'
+export SLACK_CHANNEL='#infodev-notifications'
 export targets=( "oce" "asa" "eu" "oie" "wf" "oag" )
```

Comment:
Do/should we need to add the beta targets (open and closed) as well?
Right now I'm building a closed beta, which is what reminds me of this now.

Reply:
Do we plan to translate betas? If yes, I'll add those targets later when we get there.

Build notifications are send from `info-dev` script. This script notifies when topic for translation is created